<?xml version="1.0"?>
<launch>

  <arg name="dir_name" default="/config/maps" />
  <arg name="map_name" default="map" />
  <arg name="keepout_map_name" default="keepout" />
  
  <node name="map_saver" pkg="map_server" type="map_saver"
	  args="-f $(find task_programmer)$(arg dir_name)/$(arg map_name)" output="screen"/>
  <node name="keepout_map_saver" pkg="map_server" type="map_saver"
    args="-f $(find task_programmer)$(arg dir_name)/$(arg keepout_map_name)" output="screen"/>
</launch>
